### DISCUSSION TITLE
Wrong line width for fillings in exported SVG

### DISCUSSION THREAD

**Discussion by luoyu-123** [mozman/ezdxf#310] [Q&A]
Dear Mr. Manfred Moitzi:
When I use the ezdxf plug-in to export the SVG image format, the marked arrow block in the exported image is incomplete. Is there any way to set the integrity of the arrow block?
Saved svg picture：
![image](https://user-images.githubusercontent.com/59160360/104806735-a1148980-5814-11eb-9f97-54d6bf31cb78.png)
Saved dxf file：
![image](https://user-images.githubusercontent.com/59160360/104806763-d7520900-5814-11eb-8b4b-5a4f9460d078.png)

Best wishes

**Comment by mbway** [comment]
are you able to share a dxf file that has this issue? it might make the problem easier to track down.

Have you tried saving as a png as well to see whether that makes a difference. the problem could potentially be the viewer you are viewing the svg in?

I don't think the matplotlib backend uses matplotlib arrow rendering and instead draws filled polygons so it's surprising that the arrows look slightly misplaced (which can happen in matplotlib by passing the wrong value of `length_includes_head` to `ax.arrow()` but that doesn't seem to be the issue here if `ax.arrow()` is not being used)

**Comment by mozman** [comment]
As already remarked in a previous issue: Please post your questions as issues, that's what issues for -- if your question may helpful to others I will convert the issue to discussions. The conversion from discussions to issues is yet not possible!

**Comment by luoyu-123** [comment]
Okay, this is the dxf file I want to convert
[4_11_39.txt](https://github.com/mozman/ezdxf/files/5827224/4_11_39.txt)
I changed the dxf file to a txt file, if necessary, please change the .txt suffix to .dxf suffix, thank you very much
This is the converted Python program
[dxf_to_svg.txt](https://github.com/mozman/ezdxf/files/5827242/dxf_to_svg.txt)
Please change the .txt suffix to .py if necessary, thank you very much for your help
best wishes

**Comment by luoyu-123** [comment]
In addition, saving as png format is the same, thank you very much

![image](https://user-images.githubusercontent.com/59160360/104863410-b272a880-5970-11eb-844c-c0da03636de1.png)

**Comment by mozman** [comment]
Bug in the matplotlib backend should be fixed:
![image](https://user-images.githubusercontent.com/4230966/104870561-1f724800-5949-11eb-8965-fdbd9e62c17f.png)

**Reply by luoyu-123** [reply]
Excuse me, can you give me your test code? Is it because I set the parameters incorrectly during the conversion, thank you very much
Best wishes

**Reply by mozman** [reply]
```Python
import ezdxf
from ezdxf.addons.drawing import matplotlib

doc = ezdxf.readfile('4_11_39.dxf')
matplotlib.qsave(
    doc.modelspace(), '1231313.svg', bg="#FFFFFF", backend='svg', dpi=350,
    params={
        'lineweight_scaling': 0.05,
        'min_lineweight': 0.01,
    })
```

**Reply by luoyu-123** [reply]
Thank you Mr. Manfred Moitzi
best wishes
